src/gui/file_list_defs.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "ffconf.h"
+
+/*
+ * Using limits from marlin:
+ * FILENAME_LENGTH				13
+ * LONG_FILENAME_LENGTH			13 * 5 + 1
+ * MAX_DIR_DEPTH				10
+ * MAXDIRNAMELENGTH   			8
+ * MAXPATHNAMELENGTH			(1 + (MAXDIRNAMELENGTH + 1) * (MAX_DIR_DEPTH) + 1 + FILENAME_LENGTH)
+ * */
+
+#ifndef MAX_DIR_DEPTH
+    #define MAX_DIR_DEPTH 10 // Maximum folder depth
+#endif
+
+#ifndef MAXDIRNAMELENGTH
+    #define F_MAXDIRNAMELENGTH _MAX_LFN + 1
+#endif
+
+#ifndef MAXPATHNAMELENGTH
+    #define F_MAXPATHNAMELENGTH (1 + (F_MAXDIRNAMELENGTH + 1) * (MAX_DIR_DEPTH) + 1 + _MAX_LFN)
+#endif
```

Comment:
Isn't this a mistake? (the fact that the macros in `#ifdef` don't have the `F_` prefix and the ones being defined do have it)

Reply:
(I just noticed you extracted the code from `src/gui/window_file_list.h`, but still...)

Reply:
This was reused from the original code, in fact it's been confusing already - there were and still are 2 types of macros: with "F_" and without. They look like being interchangeable, but their real values differ in some cases. This must be improved when rewriting to c++.